tests/unit/tests_standard_lib/tests_sample_generation/test_data/xmls/lr_incorrect.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<device>
+  <vendor>Microsoft</vendor>
+  <product>Sysmon</product>
+  <version id="15.0" />
+  <event code="19" name="EventID_19_WmiEvent_(WmiEventFilter_activity_detected)" format="">
+    <transport type="windows_input" sourcetype="xmlwineventlog" source="XmlWinEventLog:Microsoft-Windows-Sysmon/Operational" host="SERVER1" />
+    <NONEXISTING>HELLO</NONEXISTING>
+    <source>
+      <jira id="" />
+      <comment>lab, index = * EventCode=19</comment>
```

Comment:
Since the purpose of this element `<comment>` is only to specify the origin of the event,  should we change its name  to a more meaningful, e.g. to `<origin>`? Since we are changing the XML schema now it would be a good time to do that. Because the new element `<note>` sounds redundant to `<comment>`.

Reply:
We can do that as part of a separate PR, we will be doing some code reorganization next week. If we change the schema here - none of existing XMLs are going to be valid and that would trigger a lot of mess for developers.

Reply:
ok, sure